StRoot/macros/bfc.C

```diff
@@ -9,7 +9,11 @@
 //////////////////////////////////////////////////////////////////////////
 class StBFChain;        
 class StMessMgr;
-#if !defined(__CINT__) || defined(__MAKECINT__)
+
+#pragma cling load("StUtilities")
```

Comment:
So... I did a search for documentation on this feature of cling... and only was able to find the load pragma in their github repo under tests...

https://github.com/root-project/cling

Regardless,... this appears to solve the problem, enabling the necessary libraries to be preloaded.  I *think* you could also achieve this by loading libraries in the standard rootlogon.C.  But this would be more general (and allows individual users to implement their own root logons).

Reply:
Yes, rootlogon.C, might be possible, but, if it can work without the "secret sauce", I prefer that)